agents/rust/aries-vcx-agent/src/agent/init.rs

```diff
@@ -75,7 +85,8 @@ impl Agent {
         };
 
         let anoncreds = IndyCredxAnonCreds;
-        let (ledger_read, ledger_write) = build_ledger_components(vcx_pool_config).unwrap();
+        let (ledger_read, ledger_write) = build_ledger_components(vcx_pool_config.clone()).unwrap();
+        let (ledger_read_2, _) = build_ledger_components(vcx_pool_config).unwrap();
```

Comment:
Addressed by generalizing did_resolver_sov to work with Borrow trait bound rather than:
- owned value as I had it before
- raw reference as it was in original PR

In agent, I war ledger_read in Arc